src/containers/Article.js

```diff
@@ -174,8 +187,9 @@ class Article extends Component {
   }
 
   componentWillUnmount() {
-    window.removeEventListener('resize', this._setArticleBounding)
+    window.removeEventListener('resize', this._onResize)
```

Comment:
Since you add `throttle` here, could you kindly add throttle on resize event listener?